1_🔍_Search.py

```diff
@@ -286,10 +283,8 @@ def get_cached_response(_query_engine, _slack_client, query):
             st.error(msg)
 
 elif st.session_state.response:
-    if st.session_state.response_error:
-        st.error(st.session_state.response)
-    else:
-        st.success(st.session_state.response)
+    display_func = st.error if st.session_state.response_error else st.success
+    display_func(st.session_state.response)
```

Comment:
This version feels a bit harder to read than the original. If `display_func` were called multiple times, pulling it out might help. But since it's only used once, the extra layer of indirection might not be worth it. But good instinct to explore refactorings!